.github/workflows/ci.yml

```diff
@@ -7,6 +7,24 @@ on:
   pull_request:
 
 jobs:
+  check-format:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout"
```

Comment:
lot's of code duplication.

As I am not an expert on github CI: Can you please check if there is a better option?
E.g. defining a parent job.

Reply:
I can do something. :eyes:

Reply:
Done.